### PR TITLE
Dmitrii tofan first java track

### PR DESCRIPTION
**Pull request by Dima12A**
all done

### CODE REVIEW

**Review by dgrimaliuc — CHANGES_REQUESTED** [review]
Remove idea folder

**Comment by Dima12A** [comment]
HW_2 - done

**Review by dgrimaliuc — CHANGES_REQUESTED** [review]
Remove Idea folder

File: SimpleJavaProject/src/main/java/dmitrii_tofan/ConditionMain.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            System.out.println("Number is prime");
+        } else {
+            System.out.println("Number is even");
+        }
+//    4. Determine the grade based on a given score.
+//A > 90
+//B > 80
+//C > 70
+        int score = 25;
+        if (score >= 90) {
```
Comment (dgrimaliuc) [inline SimpleJavaProject/src/main/java/dmitrii_tofan/ConditionMain.java:34]: please try it with switch ()

File: SimpleJavaProject/src/main/java/dmitrii_tofan/Main.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package dmitrii_tofan;
```
Comment (dgrimaliuc) [inline SimpleJavaProject/src/main/java/dmitrii_tofan/Main.java:1]: In this PR should be only this file!

**Review by dgrimaliuc — CHANGES_REQUESTED** [review]

**Review by dgrimaliuc — CHANGES_REQUESTED** [review]
Please remove idea folder and
Please update your branch, 
pull last changes from master or do rebase.

**Pull request closed without merging**